db/sqldb/user.go

```diff
@@ -75,13 +75,18 @@ func (d *DB) CreateUser(tx db.Tx, u *pacta.User) (pacta.UserID, error) {
 	if err := validateUserForCreation(u); err != nil {
 		return "", fmt.Errorf("validating user for creation: %w", err)
 	}
+	var pl *string
```

Comment:
Same nit, I feel icky about pointer primitives when our SQL driver has dedicated types for this

Reply:
SG